test/ambicmd.vimspec

```diff
@@ -210,7 +212,19 @@ Describe s:get_completion()
     Assert Equals(GetCompletion('foo'), ['Foo', 0])
     Assert Equals(GetCompletion('foob'), ['FooBa', 0])
   End
+  It does not treat buffer-local command and global command as different completion candidates.
+    Assert Equals(GetCompletion('cmdbothgloballocal'), ['CmdBothGlobalLocal', 1])
+  End
   After all
+    " The `-buffer` argument for :delcommand is introduced in Vim 8.2.3433 or
+    " Neovim 0.7.0.
+    if has('patch-8.2.3433') || has('nvim-0.7.0')
+      delcommand -buffer CmdBothGlobalLocal
```

Comment:
I don't want to bother with Neovim.
Avoid this conditional branch.

1. Create a new context.
2. Create a new buffer in before section.
3. Create a buffer-local command.
4. Close a created buffer in after section.

Reply:
Fixed.  Can you review again, please?